core/search/phrase_filter.hpp

```diff
@@ -67,7 +65,8 @@ class by_phrase_options {
   // Returns reference to the inserted phrase part
   template<typename PhrasePart>
   PhrasePart& insert(PhrasePart&& t, size_t pos) {
-    is_simple_term_only_ &= std::is_same<PhrasePart, by_term_options>::value; // constexpr
+    is_simple_term_only_ &=
+        std::is_same<PhrasePart, by_term_options>::value;  // constexpr
```

Comment:
remove useless comment
compiler should optimize this

Reply:
done